### PR TITLE
Minor spelling corrections

### PR DESCRIPTION
**Pull request by davidlatapie**
The biggest one was a missing word.

### CODE REVIEW

File: translations/monero-core_fr.ts [hunk 1]
```diff
@@ -150,7 +150,7 @@
     <message>
         <location filename="../pages/History.qml" line="70"/>
         <source> selected: </source>
-        <translation> séléctionné : </translation>
+        <translation> sélectionné&nbsp;: </translation>
```
Comment (Jaqueeee) [inline translations/monero-core_fr.ts]: Any special reason for changing all spaces to &nbsp;? Also wonder if the space between the word and the : needed anyway? Look at the source string.

File: translations/monero-core_fr.ts [hunk 2]
```diff
@@ -505,7 +505,7 @@
     <message>
         <location filename="../components/SearchInput.qml" line="69"/>
         <source>Search by...</source>
-        <translation>Chercher par...</translation>
+        <translation>Chercher par…</translation>
```
Comment (Jaqueeee) [inline translations/monero-core_fr.ts]: Was this change intentional?

File: translations/monero-core_fr.ts [hunk 3]
```diff
@@ -1286,7 +1286,7 @@ Description : </translation>
     <message>
         <location filename="../main.qml" line="731"/>
         <source>Please wait...</source>
-        <translation>Veuillez patienter...</translation>
+        <translation>Veuillez patienter…</translation>
```
Comment (Jaqueeee) [inline translations/monero-core_fr.ts]: and this

File: translations/monero-core_fr.ts (hunk 1 above)
Comment (davidlatapie) [inline translations/monero-core_fr.ts]: French typography requires non-breaking spaces (and in some instance, thin non-break spaces)

File: translations/monero-core_fr.ts (hunk 2 above)
Comment (davidlatapie) [inline translations/monero-core_fr.ts]: Yes

File: translations/monero-core_fr.ts (hunk 3 above)
Comment (davidlatapie) [inline translations/monero-core_fr.ts]: Ditto. According to Unicode, ... and … convey the same exact meaning, so you can change it back if you're not comfortable with this, though. Still, I consider ellipsis to carry a semantic value dot dot dot doesn't have.

Comment (Jaqueeee) [inline translations/monero-core_fr.ts]: OK. Was just curious if it was changed by you or automatically by your text editor. :)

File: translations/monero-core_fr.ts (hunk 1 above)
Comment (Jaqueeee) [inline translations/monero-core_fr.ts]: OK. Cool. We haven't really tested the translations yet, so i'm not sure how the urlencoded strings will be treated.
